lib/src/date-input/Calendar.tsx

```diff
@@ -0,0 +1,263 @@
+        dateToFocus.get("day") !== 0
+          ? (dateToFocusTemp = dateToFocusTemp.day(1))
+          : (dateToFocusTemp = innerDate.date(date.day - 1).day(1));
+        focusDate(dateToFocusTemp);
+        break;
+      case "End":
+        event.preventDefault();
+        dateToFocusTemp.get("day") !== 0 && (dateToFocusTemp = dateToFocusTemp.day(7));
+        focusDate(dateToFocusTemp);
+        break;
+      case " ":
+        event.preventDefault();
+        onDaySelect(dateToFocusTemp);
+        break;
+    }
+  };
+  return (
+    <CalendarContainer>
+      <WeekHeader>
+        {weekDays.map((weekDay) => (
+          <WeekHeaderCell key={weekDay}>{weekDay}</WeekHeaderCell>
+        ))}
+      </WeekHeader>
+      <DayCellsContainer>
+        {dayCells.map((date, index) =>
+          date !== 0 ? (
+            <DayCell
+              onKeyDown={(event) => handleDayKeyboardEvent(event, date)}
+              aria-label={date.day}
+              id={`day_${date.day}_month${date.month}`}
+              key={`day_${index}`}
+              onClick={() => onDateClickHandler(date)}
+              selected={isDaySelected(date, selectedDate, innerDate)}
+              actualMonth={date.month === innerDate.get("month")}
+              autoFocus={date.day === dateToFocus.get("date") && date.month === dateToFocus.get("month")}
+              aria-selected={isDaySelected(date, selectedDate, innerDate)}
+              tabIndex={date.day === dateToFocus.get("date") && date.month === dateToFocus.get("month") ? 0 : -1}
+              isCurrentDay={
+                today.get("date") === date.day &&
+                today.get("month") === innerDate.get("month") &&
+                today.get("month") === date.month &&
+                today.get("year") === innerDate.get("year")
+              }
+            >
+              {date.day}
+            </DayCell>
+          ) : (
+            <EmptyDayCell key={`empty_${index}`} />
+          )
+        )}
+      </DayCellsContainer>
+    </CalendarContainer>
+  );
+};
+
+const CalendarContainer = styled.div`
+  width: ${(props) => props.theme.dateInput.pickerWidth};
+  height: 260px;
+  background: ${(props) => props.theme.dateInput.pickerBackgroundColor};
+  display: flex;
+  flex-direction: column;
+  padding: 0px 8px 8px 8px;
+  justify-content: center;
+  border-radius: 4px;
+  box-sizing: border-box;
+`;
+
+const WeekHeader = styled.div`
+  display: flex;
+  height: 36px;
+  align-items: center;
+  justify-content: center;
+`;
+
+const WeekHeaderCell = styled.span`
+  color: ${(props) => props.theme.dateInput.pickerWeekFontColor};
+  width: 36px;
+  text-align: center;
+  font-family: ${(props) => props.theme.dateInput.pickerFontFamily};
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  letter-spacing: 0.03333em;
+`;
+
+const DayCellsContainer = styled.div`
+  background: ${(props) => props.theme.dateInput.pickerBackgroundColor};
+  height: 224px;
```

Comment:
This `height: 224px` extracted from the designs is not working as expected: 

<img width="1226" alt="Sin título" src="https://user-images.githubusercontent.com/44321109/207387581-d26a9ffb-b159-4a20-9306-de02df140c4e.png">

To fix this, you just need to remove that line and add a padding-top of `8px` (the calc would be 36px * 6 rows + 8px).

<img width="182" alt="Sin título" src="https://user-images.githubusercontent.com/44321109/207388635-87bb1fe2-2ead-49fa-87b6-e3b8d4b75ae0.png">